login-workflow/src/screens/Login.tsx

```diff
@@ -334,7 +335,10 @@ export const Login: React.FC = () => {
                         label={t('pxb:LABELS.PASSWORD')}
                         className={clsx(classes.passwordFormField, { [classes.hasError]: isInvalidCredentials })}
                         value={passwordInput}
-                        onChange={(evt: ChangeEvent<HTMLInputElement>): void => setPasswordInput(evt.target.value)}
+                        onChange={(evt: ChangeEvent<HTMLInputElement>): void => {
+                            setPasswordInput(evt.target.value);
+                            authUIState.login.transitErrorMessage = null;
```

Comment:
this works, but it feels like it's probably not a good practice to directly mutate the value that we are pulling from a Context. I think a safer bet would be to dispatch the reset action which will fully clear the TransitState:
```tsx
if(authUIState.login.transitErrorMessage !== null) AccountActions.resetLogin();
```